Code/Framework/AzCore/AzCore/std/allocator_static.h

```diff
@@ -182,14 +153,10 @@ namespace AZStd
         }
 
         // When we copy the allocator we don't copy the allocated memory since it's the user responsibility.
-        AZ_FORCE_INLINE static_pool_allocator(const this_type& rhs)
-            : m_name(rhs.m_name)    {  reset(); }
-        AZ_FORCE_INLINE static_pool_allocator(const this_type& rhs, const char* name)
-            : m_name(name) { (void)rhs; reset(); }
-        AZ_FORCE_INLINE this_type& operator=(const this_type& rhs)      { m_name = rhs.m_name; return *this; }
-
-        AZ_FORCE_INLINE const char*  get_name() const           { return m_name; }
-        AZ_FORCE_INLINE void         set_name(const char* name) { m_name = name; }
+        AZ_FORCE_INLINE static_pool_allocator(const this_type&)
+            {  reset(); }
```

Comment:
nitpick: extra space before reset().
```suggestion
            { reset(); }
```

Reply:
Fixed.